.github/workflows/eamxx-sa-sanitizer.yml

```diff
@@ -0,0 +1,62 @@
+name: eamxx-sa-sanitizer
+
+on:
+  workflow_dispatch:
+
+  # Add schedule trigger for nightly runs at midnight MT (Standard Time)
+  schedule:
+    - cron: '0 7 * * *'  # Runs at 7 AM UTC, which is midnight MT during Standard Time
+
+concurrency:
+  # Two runs are in the same group if they are testing the same git ref
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  submit: ${{ github.event_name == 'schedule' && 'true' || 'false' }}  # Submit to cdash only for nightlies
```

Comment:
I didn't want to submit to cdash when manually running. If for some reason our nightlies hang or are killed, we can re-run the nightly job to manually see what's going on. It won't get uploaded to cdash, but we'll be able to see on the actions tab.

I don't have a strong preference here. I liked this approach b/c it's more concise. Making the 'submit' an input of workflow_dispatch will force the env to be determined in the run steps (since it's dependent on "dynamic" values, like `inputs`, while `event_name` is "static"), so the wf file will be a bit longer and a tad harder to parse maybe. But I can see the value.

What do other eamxx devops think? @jgfouca @tcclevenger @AaronDonahue ?